maltd-frontend/src/components/base/ListElement/ListElement.stories.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import ListElement from "./ListElement";
+
+const listElement = {
+  title: "MALTD",
+  description: "Dynamics Account Management Tool"
+};
+
+storiesOf("List Element", module)
+  .add("current projects", () => <ListElement listElement={listElement} />)
+  .add("users", () => (
+    <ListElement
+      listElement={{
+        ...listElement,
+        title: "Pat Simonson",
+        description: "psimon@gov.bc.ca"
```

Comment:
I know this is dummy data, but are there any issues with this being in the repo if it did actually exist? It already exists in the PR history now.

Dumb question clearly.

Reply:
hmm yeah I didn't check that it doesn't exist - however, I did make it up as dummy data based on the mockups - I will change it to be something more dummy though just as a safety